issues/html-wrongcaseenumvalue.md

```diff
@@ -12,4 +14,6 @@ While crawling [HTML Standard](https://html.spec.whatwg.org/multipage/), the fol
 
 See [Use casing rules consistent with existing APIs](https://w3ctag.github.io/design-principles/#casing-rules) in Web Platform Design Principles document for guidance.
 
+<sub>Cc @dontcallmedom @tidoust</sub>
```

Comment:
I note that template change explains why this and other issues were raised again even though there haven't been any substantive change. If we keep making changes to the template, we may consider trying to be smarter in the comparison. In the meantime, merging should be enough (and the issue should not appear again next time the job runs).